selectable_pdf_infra/selectable_pdf_infra/selectable_pdf_stack.py

```diff
@@ -138,15 +148,15 @@ def __init__(
             )
         )
 
+
         # Lambda getting the textract results from S3 and feeding textract output 
         # status to a SQS for future faning
-        
-        sqs_visibility_timeout_sec = 2 * 60
+        sqs_visibility_timeout_sec = 10 * 60
```

Comment:
I would capitalize all constants and put at the top of the file

Reply:
These are variables for different services, like lambda functions. There are one `sqs_timeout` queue or one `lambda_timeout` per function. Therefore these numbers are not really constants, like the region, or the account ID. For the sake of visibility, I would rather leave these variables where they are.